cafebabel/static/js/tag-detail.js

```diff
@@ -1,35 +1,20 @@
 const container = document.querySelector('.tag-header')
-const imgUrl = container.getAttribute('data-image')
+const imgUrl = container.querySelector('img')
 
 addBackgroundImage(container, imgUrl)
-parallax(imgUrl)
+parallax()
 
-function parallax(imgUrl) {
+function parallax() {
   const container = document.querySelector('.tag-header.image')
-  const height = getHeight(imgUrl)
-
   if (!container) return
   window.addEventListener('scroll', () => {
     const scrolledHeight = window.pageYOffset
-    const top = container.offsetTop - height * 2
-    container.style.backgroundPositionY = `${(top - scrolledHeight) / 3}px`
+    container.style.backgroundPositionY = `${50 + 0.2 * scrolledHeight}%`
   })
 }
 
 function addBackgroundImage(container, imgUrl) {
-  if (imgUrl) {
-    container.classList.add('image')
-    container.style.backgroundImage = `url(${imgUrl})`
-  } else {
-    container.classList.add('default')
-  }
-}
-
-function getHeight(url) {
-  const img = new Image()
-
-  img.addEventListener('load', () => img.height)
-  img.src = url
-
-  return img.height
+  if (!imgUrl) return
+  container.querySelector('img').classList.add('hidden')
```

Comment:
Why not reuse `imgUrl`?

Reply:
ok. and I changed `imgUrl` by `bgImg` => more sense, right ?

Reply:
Even better: `backgroundImage`? :)